Add Voxtral test.

shoumikhin opened the pull request:
Summary:
X-link: https://github.com/meta-pytorch/tokenizers/pull/136

.

Differential Revision: D84081392

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

shoumikhin committed:
Add Voxtral test for preprocessed audio. (#14918)

Summary:

X-link: https://github.com/meta-pytorch/tokenizers/pull/136

Also, undo previous regex modifications in tokenizers lib.

Reviewed By: larryliu0820

Differential Revision: D84081392
